src/pfs/pfs.proto

```diff
@@ -181,6 +182,8 @@ message InspectRepoRequest {
 
 message ListRepoRequest {
   reserved 1;
+  string type = 2;
+  bool all = 3;
```

Comment:
I think we can exclude `all` if we interpret an empty `type` field as 'no filter'.

Reply:
I think I was debating if it made the signature of ListReposByType confusing, but I suppose most people will just want ListRepos anyway.